src/plugins/dashboard/server/dashboard_saved_object/dashboard_saved_object.ts

```diff
@@ -38,36 +40,52 @@ export const createDashboardSavedObjectType = ({
     },
   },
   mappings: {
+    dynamic: false,
     properties: {
       description: { type: 'text' },
-      hits: { type: 'integer', index: false, doc_values: false },
-      kibanaSavedObjectMeta: {
-        properties: { searchSourceJSON: { type: 'text', index: false } },
-      },
-      optionsJSON: { type: 'text', index: false },
-      panelsJSON: { type: 'text', index: false },
-      refreshInterval: {
-        properties: {
-          display: { type: 'keyword', index: false, doc_values: false },
-          pause: { type: 'boolean', index: false, doc_values: false },
-          section: { type: 'integer', index: false, doc_values: false },
-          value: { type: 'integer', index: false, doc_values: false },
-        },
-      },
-      controlGroupInput: {
-        properties: {
-          controlStyle: { type: 'keyword', index: false, doc_values: false },
-          chainingSystem: { type: 'keyword', index: false, doc_values: false },
-          panelsJSON: { type: 'text', index: false },
-          ignoreParentSettingsJSON: { type: 'text', index: false },
-        },
-      },
-      timeFrom: { type: 'keyword', index: false, doc_values: false },
-      timeRestore: { type: 'boolean', index: false, doc_values: false },
-      timeTo: { type: 'keyword', index: false, doc_values: false },
       title: { type: 'text' },
-      version: { type: 'integer' },
```

Comment:
Removing `version` will be a breaking change because the field is indexed.

Reply:
Thank you for the information @TinaHeiligers! Is it okay to remove mappings that are `index: false`?

Reply:
We discussed removing mappings offline within the Core team and we'd prefer not to risk it. Removing mappings may not seem like a huge change but it would mean either another migration or be released untested.

Reply:
@TinaHeiligers thank you for the information! I will close this PR for the time being, and add in the schema changes as part of my content management onboard PR.